src/wasm-lib/tests/executor/main.rs

```diff
@@ -1328,11 +1328,7 @@ const bracket = startSketchOn('XY')
 "#;
 
     let result = execute_and_snapshot(code, UnitLength::Mm).await;
-    assert!(result.is_err());
-    assert_eq!(
-        result.err().unwrap().to_string(),
-        r#"engine: KclErrorDetails { source_ranges: [SourceRange([1329, 1430])], message: "Modeling command failed: [ApiError { error_code: BadRequest, message: \"Fillet failed\" }]" }"#
-    );
+    assert!(!result.is_err());
```

Comment:
This is giving me a lint warning.

```suggestion
    assert!(result.is_ok());
```

Reply:
yep, already handled it locally.  i'm backed up on deployment of engine (separate fix needed for a test here) before I push